### PR TITLE
Update versions of dependencies, drop the very old `typings` tool.

### PR DESCRIPTION
**Pull request by rictic**
This is a major breaking change, so will go out as @Polymer/tools-common@2.0.0

### CODE REVIEW

File: gulp-tasks.js [hunk 1]
```diff
@@ -30,11 +29,7 @@ function task(name, deps, impl) {
 }
 
 module.exports.init = function() {
-  task('init', () => {
-    if (fs.existsSync('./typings.json')) {
-      gulp.src('./typings.json').pipe(typings())
-    }
-  });
+  task('init', () => {});
```
Comment (justinfagnani) [inline gulp-tasks.js]: just remove?

Comment (rictic) [reply gulp-tasks.js]: +1, done

File: .eslintrc.json [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
-  "ecmaVersion": 6,
   "env": {
     "mocha": true,
     "node": true,
     "es6": true
   },
   "rules": {
-    "comma-dangle": [2, "always-multiline"],
-    "no-unused-vars": [2, { "vars": "all", "args": "none" }],
+    "comma-dangle": [
```
Comment (FredKSchott) [inline .eslintrc.json:9]: is there any situation where we still use eslint vs. clang or tslint?

**Review by FredKSchott — APPROVED** [review]

File: .eslintrc.json (hunk 2 above)
Comment (rictic) [reply .eslintrc.json:9]: A few I think? I'd like to trop those cases down to zero, but we're not there yet I don't think.

**Pull request merged**